src/core/edit/edit.model.js

```diff
@@ -10,5 +10,11 @@ export default class EditModel {
 		this.commit = new Command();
 		this.cancel = new Command();
 		this.reset = new Command();
+		this.commitShortcuts = {
+			'$default': 'tab|enter',
```

Comment:
`ctrl+s` is a common system shortcut, so we I don't think we need to intercept this